app.js

```diff
@@ -167,6 +167,34 @@ app.post("/rideInfo", function(req, resp){
 				});
 })
 
+app.put("/rideInfo", function(req, resp){
+
+    if(req.query.rideId){
+        var rideStatus;
+
+        if(req.query.status){
+          rideStatus = req.query.status;
+        };
+
+        if(req.query.analysisStatus){
+          analysisStatus = req.query.analysisStatus;
+        };
+
+				mongoose.model('rideInfo').findOneAndUpdate({ rideId : req.query.rideId}, { status : req.query.status }, function (err, rideInfo) {
```

Comment:
Looks like this code allows me to update only the `status`. We would also want to update other properties of rideInfo - `startedAt`, `endedAt` etc. Depending on the properties set in the query params, update the respective fields. 

After making changes update this pull request.